### DISCUSSION TITLE
Controls not working, PS Vita

### DISCUSSION THREAD

**Discussion by nikolanii** [LizardByte/community#94] [Sunshine] [answered]
Hello!

I tested Sunshine on my Vita, it does not recognize the controls nicely, it only recognizes the touch screen somehow.
I set the controller type to be "XBOX 360", that was the default setting, no luck, changed it to DS4, no luck..
Windows simply does not recognize vita as a controller in any way.

Everything works properly through NVIDIA GeForce Experience

Is there a way to fix it?

EDIT: I'm stupid, just now saw a similar question, sorry!

**Comment by ReenigneArcher** [comment, marked as answer]
> EDIT: I'm stupid, just now saw a similar question, sorry!

I was going to say...

https://github.com/LizardByte/Sunshine/discussions/484
